### ISSUE TITLE
Russian/Japanese/Chinese localization is terribly broken on Windows

### ISSUE THREAD

**Issue by xyzz** [minetest/minetest#910]
![screenshot - 090713 - 23 12 04](https://f.cloud.github.com/assets/1065521/1103620/ef3e06e4-18b7-11e3-833a-3466966091fe.png)

**Comment by Fixer-007** [comment]
Not fixed in 0.4.14. Please reopen.

```
[21:17:30] <minetestusr> hi, i have a problem in windows 10 latest stable build, no text in menus
[21:20:04] <minetestusr> i have text in the server names, etc..
[21:20:18] <minetestusr> but all the tabs / and the first window are textless
[21:21:34] <minetestusr> the windows language is english, but I have hebrew enabled
[21:21:40] <minetestusr> and it's the non-english locale
[21:22:52] <minetestusr> ok, fixed the issue
[21:22:57] <minetestusr> i deleted locale/he and everything is ok
[21:25:15] <minetestusr> found the fix here https://forum.minetest.net/viewtopic.php?f=6&t=7394
[21:25:19] <minetestusr> 0.4.14
```

**Comment by Fixer-007** [comment]
Did some research on locales:
he (no text), ko (no text on some buttons), ky (few missings symbols?)
